fedimint-server/src/consensus/mod.rs

```diff
@@ -271,38 +271,39 @@ impl FedimintConsensus {
                 )
                 .expect("DB Error");
                 dbtx.remove_entry(&ProposedTransactionKey(transaction.tx_hash()))
+                    .await
                     .expect("DB Error");
             }
 
             let caches = self.build_verification_caches(ok_tx.iter());
             for transaction in ok_tx {
-                let span = info_span!("Processing transaction");
```

Comment:
Created a thread on discord about this. I'm not sure why its necessary, but I needed to remove it to add an await to remove_entry.

Reply:
Updated this to use the async version of span https://docs.rs/tracing/latest/tracing/struct.Span.html#in-asynchronous-code